client/shared/src/main/scala/org/http4s/client/websocket/WSClient.scala

```diff
@@ -219,8 +259,8 @@ private[http4s] object WSClient {
               }
             defrag(Chain.empty, ByteVector.empty).value
           }
-          override def subprocotol: Option[String] = conn.subprotocol
-          override def closeFrame: Deferred[F, WSFrame.Close] = recvCloseFrame
+          override def subprotocol: Option[String] = conn.subprotocol
```

Comment:
Wait, why does this not throw an error? I feel like I am missing sth here, Scala 2.13.8 rejects this (without any extra flags):

```scala
trait A { def a: Int = 1 }
new A { override def b: Int = 2 }

```

Reply:
What error are you expecting it to throw?

Reply:
Ahhh, I somehow missed that "subprotocol" was misspelled in both the definition and here, somehow I assumed it was only case here. That left me very confused why it did not throw an "overrides nothing" error 😆